src/web/components/SignInGate/gateDesigns/SignInGateDismissWindow.tsx

```diff
@@ -0,0 +1,268 @@
+import React, { useState, Suspense } from 'react';
```

Comment:
Do we need a whole separate gate design if it's the same as the previous design, and just re use that?

Reply:
in cypress we can reuse the same `data-cy` selectors that main uses, and in the files we set up the variants we can just the main design too

Reply:
as that's all i can see thats different?

Reply:
Yeah, I did this originally then decided to just dupe the design but can switch back to using main if you prefer.

Reply:
Yeah I think it's better we reuse as much as possible without duping.
I'm thinking instead of `SignInGateMainDesign` we should call is `SignInGateViiDesign` to match the name in the sheet?

Reply:
done. you mean rename the design file back to vii-variant?

Reply:
Yeah that makes sense. Thanks.